less/v1/apps.less

```
@@ -602,6 +602,18 @@ dnd5e-effects {
   }
 }
 
+.damage-modification {
+  .modifications label {
```

This may be an issue in all the trait selectors, but from the screenshot, it looked like this could do with some spacing between sections:

```suggestion
  .modifications {
    padding-bottom: 6px;
    margin-bottom: 6px;
  }
  .modifications label {
```